vmdk_plugin/utils/plugin_utils/plugin_utils.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugin_utils
+
+// This file holds utility/helper methods required in plugin module
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+const (
+	linuxMountsFile = "/proc/mounts"
+)
+
+// GetMountInfo - return a map of mounted volumes and devices
+func GetMountInfo(mountRoot string) (map[string]string, error) {
+	volumeMap := make(map[string]string)
```

Comment:
Please add a comment of what is mapped to what (even if it's an old code). Something like `// maps volume->fullMountPath` or whatever it is doing